cfn-resources/cfn-publish.sh

```diff
@@ -1,31 +1,26 @@
 #!/usr/bin/env bash
 
-
-#trap "exit" INT TERM ERR
-#set -x
-#set -o errexit
+set -xe
 set -o nounset
-#set -o pipefail
 
 
 resources="${1:-project}"
 otherParams="${2:-}"
 
 
 if [ -n "${otherParams}" ]; then
-paramKeys=$(echo $otherParams | jq -c -r 'keys[]' | tr '\n' ' ')
+paramKeys=$(echo "$otherParams" | jq -c -r 'keys[]' | tr '\n' ' ')
 echo "Exporting the following keys..."
-echo $paramKeys
+echo "$paramKeys"
 for param in ${paramKeys};
 do
 paramKey="${param}="
-paramValue=$(echo $otherParams | jq -c -r --arg key $param '.[$key]')
-exportString=$(echo "$paramKey$paramValue")
-export "${exportString}"
+paramValue=$(echo "$otherParams" | jq -c -r --arg key "$param" '.[$key]')
+exportString="$paramKey$paramValue"
+export exportString
 echo
 done
```

Comment:
since you are here can you fix this identation

Reply:
Updated. I added [mvdan.cc/sh/v3/cmd/shfmt@latest ](https://github.com/mvdan/sh)to the devtolls and indented all the sh files. I will add this tool to the cli as well xD